CRM/Core/Payment.php

```diff
@@ -609,6 +609,22 @@ public function getText($context, $params) {
         }
         return ts('Make Contribution');
 
+      case 'eventContinueTest':
+        // This ts construct has been left with the interpolation of the button name to avoid translations having to be re-done.
```

Comment:
Shouldn't this be "Text"?

Reply:
@mattwire you mean instead of `ts` - I was referring to the fact we are still interpolating into the ts so as not to break translations